lib/matplotlib/widgets.py

```diff
@@ -1938,10 +1939,28 @@ class SpanSelector(_SelectorWidget):
 
     span_stays : bool, default: False
         If True, the span stays visible after the mouse is released.
+        Deprecated, use interactive instead.
+
+    interactive : bool, default: False
+        Whether to draw a set of handles that allow interaction with the
+        widget after it is drawn.
 
     button : `.MouseButton` or list of `.MouseButton`
         The mouse buttons which activate the span selector.
 
+    line_props : dict, default: None
```

Comment:
Since there's no canonical line in a span. the name does not make it clear what this controls. I'd call these `handle_props` or `handle_line_props` if you want to be very explicit.